commit ac1c2f44a7857c1864173d37984817be2ce081f0 (refs/changes/76/1776/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-10T11:57:51-07:00 (2 years, 6 months ago)
    
    TOOLS-1751 node-sdc-clients test failure: cnapi.test.js failures in "capacity 1" and "capacity 2" test cases
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/test/cnapi.test.js b/test/cnapi.test.js
index 58e8e46..c7d28c3 100644
--- a/test/cnapi.test.js
+++ b/test/cnapi.test.js
@@ -330,17 +330,17 @@ test('cnapi', function (tt) {
     tt.test(' capacity 1', function (t) {
         var headers = { 'x-request-id': 12345 };
 
-        cnapi.capacity([SERVER], { headers: headers }, function (err, res) {
+        cnapi.capacity([SERVER], { headers: headers }, function (err, body) {
             t.ifError(err);
-            t.equal(typeof (res), 'object');
-            t.equal(typeof (res.capacities), 'object');
-            t.equal(typeof (res.errors), 'object');
-            t.equal(Object.keys(res.capacities).length, 1);
+            t.equal(typeof (body), 'object', 'body');
+            t.equal(typeof (body.capacities), 'object', 'body.capacities');
+            t.equal(Object.keys(body.capacities).length, 1,
+                'exactly one body.capacities entry');
 
-            var server = res.capacities[SERVER];
-            t.equal(typeof (server.cpu), 'number');
-            t.equal(typeof (server.ram), 'number');
-            t.equal(typeof (server.disk), 'number');
+            var server = body.capacities[SERVER];
+            t.equal(typeof (server.cpu), 'number', 'server.cpu');
+            t.equal(typeof (server.ram), 'number', 'server.ram');
+            t.equal(typeof (server.disk), 'number', 'server.disk');
 
             t.end();
         });
@@ -348,18 +348,18 @@ test('cnapi', function (tt) {
 
 
     tt.test(' capacity 2', function (t) {
-        cnapi.capacity(null, function (err, res) {
+        cnapi.capacity(null, function (err, body) {
             t.ifError(err);
-            t.equal(typeof (res), 'object');
-            t.equal(typeof (res.capacities), 'object');
-            t.equal(typeof (res.errors), 'object');
-            t.ok(Object.keys(res.capacities).length >= 1);
-
-            Object.keys(res.capacities).forEach(function (serverUuid) {
-                var server = res.capacities[serverUuid];
-                t.equal(typeof (server.cpu), 'number');
-                t.equal(typeof (server.ram), 'number');
-                t.equal(typeof (server.disk), 'number');
+            t.equal(typeof (body), 'object', 'body');
+            t.equal(typeof (body.capacities), 'object', 'body.capacities');
+            t.ok(Object.keys(body.capacities).length >= 1,
+                'at least one body.capacities');
+
+            Object.keys(body.capacities).forEach(function (serverUuid) {
+                var server = body.capacities[serverUuid];
+                t.equal(typeof (server.cpu), 'number', 'server.cpu');
+                t.equal(typeof (server.ram), 'number', 'server.ram');
+                t.equal(typeof (server.disk), 'number', 'server.disk');
             });
 
             t.end();
